bin/spec-configs/accel-2023/runPar

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 set -x
-#export AOMP=/COD/LATEST/aomp
-export AOMP=/COD/2023-07-20/aomp
-#export AOMP=~/rocm/aomp_17.0-25
-#export AOMP=~/rocm/rocm-5.7.0-12380-npsdb-3715/llvm
 
-set +x
+export AOMP=${AOMP:-`ls -d /opt/rocm-*/llvm |sort|tail -1`}
+
+export GFX=${AOMP_GPU:-`rocm_agent_enumerator | grep -v gfx000|uniq`}
+echo "Detected gfx=$GFX"
```

Comment:
Suggest replacing all occurrences of GFX with AOMP_GPU since that is what is used in all our other scripts